### ISSUE TITLE
Axis Label Text rotation is off

### ISSUE THREAD

**Issue by Keno** [JuliaPlots/Makie.jl#754]
```
using Makie
using Zygote
f(x) = 0.1x^3 + 0.9x^2- x + 5
let scene=Scene()
	pts = -10:0.1:10
	line = Point.(f'.(pts), f.(pts))
	lines!(scene, line)
	ylabel!(scene, "y")
	xlabel!(scene, "Ty")
	display(scene)
end
```
![Screenshot from 2020-11-14 22-18-57](https://user-images.githubusercontent.com/1291671/99162083-78163200-26c7-11eb-88a5-bb9e689bd411.png)

**Comment by ffreyer** [comment]
There was an issue like this if you use CairoMakie and then switch to GLMakie. That should've been fixed with CairoMakie v0.3.5. Did you use CairoMakie in the same session?

**Comment by Keno** [comment]
Hmm, I updated something and it looks like this went away. Don't think I updated CairoMakie, but maybe some cache somewhere got refreshed.
